assets/js/components/ClustersList.jsx

```diff
@@ -24,6 +25,26 @@ const getClusterTypeLabel = (type) => {
   }
 };
 
+const getSapSystemLinkBySID = (
+  applicationInstances,
+  databaseInstances,
+  sid
+) => {
+  const foundInstance = applicationInstances
```

Comment:
I see this pattern of concatenating the app and db instances repeated.
Maybe we could create a specific selector for it.
It would be more reusable and cleaner I would say